core/src/test/java/io/aiven/klaw/controller/UiConfigControllerTest.java

```diff
@@ -204,7 +204,7 @@ public void addNewEnvMoreThan10CharsFailure() throws Exception {
     EnvModel env = utilMethods.getEnvListToAdd().get(0);
     env.setName("ABCDEFGHIJKL"); // > 10 chars, not allowed
     String jsonReq = OBJECT_MAPPER.writer().writeValueAsString(env);
-    ApiResponse apiResponse = ApiResponse.builder().message(ApiResultStatus.FAILURE.value).build();
+    ApiResponse apiResponse = ApiResponse.ok(ApiResultStatus.FAILURE.value);
```

Comment:
i think this should be notOK with a Failure result.

Reply:
good catch
i also added `ApiResponse.FAILURE`
for that case